src/App.tsx

```diff
@@ -15,35 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'alphabet',
+  Length = 'length',
+  Default = '',
+}
+
+type SortBy = {
+  sortBy: SortType | string;
```

Comment:
I would add, why do you need `| string` or even `| ''`?

You already have all valid sort cases covered in `SortType` enum